Userland/Applications/Calculator/main.cpp

```diff
@@ -68,17 +67,41 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     }));
 
     auto& round_menu = window->add_menu("&Round");
-    round_menu.add_action(GUI::Action::create("&0", [&](auto& action) {
+    GUI::ActionGroup preview_actions;
+
+    auto round_0 = GUI::Action::create_checkable("&0", [&](auto& action) {
         widget->set_rounding_length(action.text().substring(1).to_uint().value());
-    }));
+    });
 
-    round_menu.add_action(GUI::Action::create("&2", [&](auto& action) {
+    auto round_2 = GUI::Action::create_checkable("&2", [&](auto& action) {
         widget->set_rounding_length(action.text().substring(1).to_uint().value());
-    }));
+    });
+    round_2->activate(round_2);
 
-    round_menu.add_action(GUI::Action::create("&4", [&](auto& action) {
+    auto round_4 = GUI::Action::create_checkable("&4", [&](auto& action) {
         widget->set_rounding_length(action.text().substring(1).to_uint().value());
-    }));
+    });
+
+    auto round_custom = GUI::Action::create_checkable("&Custom: 0", [&](auto& action) {
```

Comment:
Please use `Custom...` here as clicking it opens a new dialog.

Reply:
Is the mechanism to display the actual rounding value a good idea, or should I remove it ?

Reply:
assuming that clicking the option opens a dialog anyway, not sure what purpose the display serves tbh

Reply:
It's currently the only way to know the value of the depth if you are in custom mode without opening the dialog. Which is, IMO not optimal

Reply:
I changed `Schrink...` too.
Is it alright for you ?

Reply:
Yes, nice!